### DISCUSSION TITLE
Code not parsed in 'Text' widget's text editor

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#861] [Q&A] [answered]
Hi,
Could you please confirm that when I input raw HTML code into the 'Text' widget's text editor and then switch to WYSIWYG mode, the HTML code is displayed as plain text and not interpreted as HTML elements?

Thanks,
Lorenzo

![2024-11-04 20 08 54](https://github.com/user-attachments/assets/8171fc54-6be4-40fc-bea9-2451c9bd1da0)

**Comment by Banuk1616** [comment]
I can confirm that, it's the same for me.

**Comment by Dnsftl** [comment]
That's right, we are experiencing the same problem

**Comment by Terresco** [comment]
Same for me

**Comment by iorbita** [comment]
... any fix for this bug?

**Reply by sonvnn** [reply, marked as answer]
This is a pain-point of Quill Editor until now. While waiting for a solution to this problem you can use Raw HTML Widget or CodePen Widget as an alternative.
![Screen Shot 2024-11-14 at 14 39 07](https://github.com/user-attachments/assets/61b94c6d-1f44-458c-b6b0-9d95328ddfa2)

You can download v3-branch to test the Raw HTML widget now.

**Reply by iorbita** [reply]
...works fine, thanks Sonny ;)
